.helm/deploy-to-cluster.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR=$(realpath "$(dirname "$0")")
+REPO_DIR=$(realpath "$SCRIPT_DIR"/..)
+
+source "$SCRIPT_DIR"/.env
+
+if [ -z "$version" ] \
+    || [ -z "$instance_name" ] \
+    || [ -z "$POSTGRES_URL" ] \
+    || [ -z "$POSTGRES_ADMIN_URL" ] \
+    ; then
+    echo "Please specify the needed env variables in $SCRIPT_DIR/.env"
+    echo "An example can be seen here:"
+    cat $SCRIPT_DIR/.env-example
+    exit 1
+fi
+
+pull_policy="Always"
+domain="ecamp3.ch"
+values="--set imageTag=${version}"
+migrations_dir="dev-data"
+
+#generated values
+app_secret=$(uuidgen)
+app_jwt_passphrase=$(uuidgen)
+app_jwt_public_key=$(echo -n "$app_jwt_passphrase" | openssl genpkey -out "$SCRIPT_DIR"/private.pem -pass stdin -aes256 -algorithm rsa -pkeyopt rsa_keygen_bits:4096)
+app_jwt_private_key=$(echo -n "$app_jwt_passphrase" | openssl pkey -in "$SCRIPT_DIR"/private.pem -passin stdin -out "$SCRIPT_DIR"/public.pem -pubout)
+
+#secrets POSTGRES_URL, POSTGRES_ADMIN_URL and sentry dsn are in .env
+
+#use 1 2 instead of "1" to deploy multiple deployments
+for i in 1; do
+  values="$values --set imageTag=${version}"
+  values="$values --set sharedCookieDomain=.$domain"
+  values="$values --set termsOfServiceLinkTemplate=https://ecamp3.ch/{lang}/tos"
+  values="$values --set api.domain=api-$instance_name-"$i".$domain"
+  values="$values --set frontend.domain=app-$instance_name-"$i".$domain"
+  values="$values --set print.domain=print-$instance_name-"$i".$domain"
+  values="$values --set mail.dummyEnabled=true"
+  values="$values --set mail.domain=mail-$instance_name-"$i".$domain"
+  values="$values --set postgresql.enabled=false"
+  values="$values --set postgresql.url=$POSTGRES_URL/ecamp3$instance_name-"$i"?sslmode=require"
+  values="$values --set postgresql.adminUrl=$POSTGRES_ADMIN_URL/ecamp3$instance_name-"$i"?sslmode=require"
+  values="$values --set postgresql.dropDBOnUninstall=true"
```

Comment:
Somehow, the `dropDBOnUninstall` option never worked for me to create/delete databases automatically on Digitalocean. The db-create job always fails with:

```
database "postgres" does not exist
```

When I create an empty database names "postgres", things worked. Did you experience the same?

Reply:
Ah, you're using the normal dev cluster and dev DB? I see we have a database named "postgres" in our dev DB, so someone might have already run into the same issue. @carlobeltrame maybe?

Reply:
I don't really remember whether I created that database. But I was under the impression that this database is the place where postgres stores internal configuration etc.? Hence why the db create / db drop commands want to work with this db, in order to insert or delete some metdadata. I'm not sure how the postgres dbms can work at all without the postgres db. But maybe I misunderstand the meaning of this database.

Reply:
Seems there is [no specific purpose of this database](https://stackoverflow.com/a/49200164), but something in our chain expects it to exist.

Digitalocean seems not to create one. A new database server comes only with a database named `defaultdb` but without one named `postgres`. Nothing tragic, but something to keep in mind when we create/recreate new devel clusters.

Reply:
Ah, I think I remember. For creating and dropping databases for deployments, we use an admin account with postgres as its username. However, in order for this user to log in, a database is required, and if you specify none then it tries to use a database with the same name as the admin account itself. So I must have taken the path of least resistance and just created this postgres db. Maybe we can specify defaultdb in the trigger instead? Have to check whether the admin user still has all necessary permissions if we do that.